### PR TITLE
Issue#281 fixed. Added Meme Generator project.

### PR DESCRIPTION
**Pull request by ParnaRoyChowdhury777**
Closes:  #281

- **Title** : Meme Generator
- **Name:** : Parna Roy Chowdhury
- **Idenitfy yourself:** : GSSOC'24 Contributor

### Describe the add-ons or changes you've made 📃

I have added the Meme Generator Project under the Intermediate Section of Vanilla Js projects along with all the requisites including the Readme.md and the Database.

### How Has This Been Tested? ⚙️

I have checked the repository and have also tested on the local machine and the project is reflecting there.

### Screenshots 📷

Screenshot 2024-06-15 164609

### Type of change ☑️

What sort of change have you made:

- [x] New feature (non-breaking change which adds functionality)

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

File: Vanilla-JS-Projects/README.md
Comment (Avdhesh-Varshney) [inline Vanilla-JS-Projects/README.md:1]: for this docs change, you have to discuss to me first or raise an issue for it.

**Review by Avdhesh-Varshney — COMMENTED** [review]
@ParnaRoyChowdhury777 this pr can't approved because,
- you have not create your own branch for raising the pr.
- doing un-necessary changes without discussion.

You can join project discord server for further communication.

File: Vanilla-JS-Projects/README.md
Comment (ParnaRoyChowdhury777) [reply Vanilla-JS-Projects/README.md:1]: What should I do now Sir, please say. I will do it accordingly

Comment (Avdhesh-Varshney) [reply Vanilla-JS-Projects/README.md:1]: either solve it by watching other merged prs of same tech stacks or if you have any more doubt, go through guidelines or join me on discord.

Comment (ParnaRoyChowdhury777) [reply Vanilla-JS-Projects/README.md:1]: Ok I will do it. Thank you for your consideration.

**Pull request closed without merging**